run_faster_whisper.py

```diff
@@ -0,0 +1,55 @@
+from faster_whisper import WhisperModel
+from segment import segment_audio, segment_video
+from embed import load_model, embed_audio
+#from embed import embed_audio
+import tensorflow as tf
+import torchaudio, torch
+import os
+from WavLM import WavLM, WavLMConfig
+
+#REMINDER: using the .local faster_whisper
+
+model_size = "large-v3"
+
+# Run on GPU with FP16
+#model = WhisperModel(model_size, device="cuda", compute_type="float16")
+
+# or run on GPU with INT8
+# model = WhisperModel(model_size, device="cuda", compute_type="int8_float16")
```

Comment:
You should probably look into using these (going back to the more resources available thing I mentioned in a different comment)